__tests__/unit/media-play-button.spec.js

```diff
@@ -0,0 +1,145 @@
+import { spy } from 'sinon';
+import { fixture, expect } from '@open-wc/testing';
+import { MediaUIAttributes, MediaUIEvents } from '../../src/js/constants';
+import { verbs } from '../../src/js/labels/labels';
+import '../../src/js/media-play-button';
+
+describe('<media-play-button/>', () => {
+  it('passes the a11y audit', async () => {
+    const el = await fixture(`<media-play-button></media-play-button>`);
+    await expect(el).shadowDom.to.be.accessible();
+  });
+
+  describe('default/playing', () => {
+    let el;
+
+    beforeEach(async () => {
+      el = await fixture(`<media-play-button></media-play-button>`);
+    });
+
+    afterEach(() => {
+      el = undefined;
+    });
+
+    it('should expect pause actions', async () => {
+      expect(el.getAttribute('aria-label')).equals(verbs.PAUSE());
+    });
+
+    it('should show pause', () => {
+      const icon = el.shadowRoot.querySelector('slot[name="pause"] > *');
+      expect(icon).to.be.visible;
+    });
+
+    /** @TODO Figure out if this test failure is due to assertion assumptions of visible vs. invalid component initialization condition (CJP) */
+    it.skip('should not show play', () => {
+      const icon = el.shadowRoot.querySelector('slot[name="play"] > *');
+      expect(icon).to.not.be.visible;
+    });
+
+    it('should request pause when clicked', () => {
+      const requestHandler = spy();
+      el.addEventListener(MediaUIEvents.MEDIA_PAUSE_REQUEST, requestHandler);
+      el.click();
+      expect(requestHandler.calledOnce).to.be.true;
+    });
+
+    it('should not request play when clicked', () => {
+      const requestHandler = spy();
+      el.addEventListener(MediaUIEvents.MEDIA_PLAY_REQUEST, requestHandler);
+      el.click();
+      expect(requestHandler.called).to.be.false;
+    });
+
+    it.skip('should request pause when Enter pressed', () => {
+      /** @TODO Implement me (CJP) */
+      expect(false).to.be.true;
+    });
+
+    it.skip('should request pause when Space pressed', () => {
+      /** @TODO Implement me (CJP) */
+      expect(false).to.be.true;
+    });
+  });
```

Comment:
Maybe just remove the ones that are skipped rn? I generally feel like having `@TODO` littered around source + test files is an anti-pattern because they're not very visible and often times never get done.

If it's *truly* a TODO -- file an issue, if not, then just remove the clutter.

That's my opinion, anyway.